frontend/scss/home.scss

```diff
@@ -2,9 +2,7 @@ body {
     margin: 0;
     padding: 0;
     overflow-x: hidden;
-    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+    font-family: "eina_regular";
```

Comment:
You defined `$font-family-sans-serif` scss variable, now you can use it here!
`font-family: $font-family-sans-serif`